src/connectors/breadcrumb/connectBreadcrumb.ts

```diff
@@ -182,7 +184,7 @@ const connectBreadcrumb: BreadcrumbConnector = function connectBreadcrumb(
 
       getWidgetRenderState({ helper, createURL, results, state }) {
         function getItems() {
-          if (!results) {
+          if (!results || state.hierarchicalFacets.length === 0) {
```

Comment:
We can add a comment that explains that this expression is required for flavors that render immediately with empty results, without relying on `init()` (like RISHooks).